index.html

```diff
@@ -19,30 +19,44 @@ <h1>Daniel Eicher</h1>
     </nav>
   </header>
   <figure>
-    <img src="https://source.unsplash.com/random/600x300" alt="Beispielbild">
-    <figcaption>Dies ist ein Beispielbild</figcaption>
+    <img src="https://source.unsplash.com/random/600x300" alt="">
+    <figcaption>This is an example image</figcaption>
   </figure>
+  <section>
+    <h2>Interests</h2>
+    <ul>
+      <li>Family</li>
+      <li>Technology</li>
+      <li>Nature</li>
+    </ul>
+  </section>
+  <section>
+    <h2>Hobbies</h2>
+    <ul>
+      <li>Hiking</li>
+      <li>Coding</li>
+      <li>Reading books</li>
+    </ul>
+  </section>
   <section id="contact">
     <h2>Contact</h2>
     <ul>
       <li><b>Phone: </b>+34 642 104 722</li>
-      <li><b>Mail: </b>danieleicher@gmx.net</li>
+      <li><b>Mail: </b><a href="mailto:danieleicher@gmx.net">danieleicher@gmx.net</a></li>
+      <li><b>Connect with me on:<br><b>
```

Comment:
Cool list and a good idea with the pictures 👍  Just one question: what is your intention with the `b` tag? If you want to have the text bold and you want to use it more then once in your code a `span` tag with a class might be the better choice ;) good work so far

Reply:
Thank you. Yes - later I will create a bold class and style it with css :)